### PR TITLE
Dw ml pipeline

### PR DESCRIPTION
**Pull request by dmwhang**
Can run test scripts to generate incidents and populate an example mysql db locally

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -41,6 +41,29 @@ python3 server.py
...
+
+```
+source /Users/agatha/panoptes/backend/database/generate_test_database.sql
+```
+
+### 2) Populate test database
+In order to populate the local mysql database run the `incident_generator.py` to generate the `incidents.txt` file. Then run the `generate_example_sql.py` file to generate the `populate_test_database.sql` file.
+
+```
+$ python3 incident_genrator.py
```
Comment (JamesTann) [inline backend/README.md:58]: Typo?

File: backend/computer_vision/daemon_test.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# import os
```
Comment (JamesTann) [inline backend/computer_vision/daemon_test.py:1]: Was this file meant to be committed?

**Review by JamesTann — APPROVED** [review]
I would check to make sure all of these files need to be committed, for example empty requirements.txt in computer vision.

File: backend/computer_vision/daemon_test.py (hunk 2 above)
Comment (dmwhang) [reply backend/computer_vision/daemon_test.py:1]: yeah but ill remove this and the empty files b4 merge

File: backend/README.md (hunk 1 above)
Comment (dmwhang) [reply backend/README.md:58]: what would be the typo?

**Review by theodorewahle — APPROVED** [review]
This looks great. I added a couple minor tweaks based on bugs I found while going through the README. Awesome besides that. I will merge.

**Pull request merged**